# Adjust the path to the hsqldb.jar file for the demo applications.
--- hsqldb-1.8.0.0.orig/demo/localhost_clientserver.html
+++ hsqldb-1.8.0.0/demo/localhost_clientserver.html
@@ -2,7 +2,7 @@
 <head><title>HSQL Database Engine</title></head>
 <body bgcolor=#FFFFFF>
 <h2>HSQL Database Engine Client/Server Demo</h2>
-<applet code="org.hsqldb.util.QueryTool" archive="hsqldb.jar" width="460" height="400" codebase="../lib/">
+<applet code="org.hsqldb.util.QueryTool" archive="hsqldb.jar" width="460" height="400" codebase=".">
 <param name="database" value="http://localhost">
 </applet>
 
--- hsqldb-1.8.0.0.orig/demo/memorydbmanager.html
+++ hsqldb-1.8.0.0/demo/memorydbmanager.html
@@ -2,7 +2,7 @@
 <head><title>HSQL Database Engine</title></head>
 <body bgcolor=#FFFFFF>
 <h2>In-Memory Applet: Database Manager</h2>
-<applet code="org.hsqldb.util.DatabaseManager" archive="hsqldb.jar" width="0" height="0" codebase="../lib/">
+<applet code="org.hsqldb.util.DatabaseManager" archive="hsqldb.jar" width="0" height="0" codebase=".">
 <param name="database" value=".">
 </applet>
 <p>
--- hsqldb-1.8.0.0.orig/demo/memoryquery.html
+++ hsqldb-1.8.0.0/demo/memoryquery.html
@@ -2,7 +2,7 @@
 <head><title>HSQL Database Engine</title></head>
 <body bgcolor=#FFFFFF>
 <h2>In-Memory Applet: QueryTool</h2>
-<applet code="org.hsqldb.util.QueryTool" archive="hsqldb.jar" width="460" height="400" codebase="../lib/">
+<applet code="org.hsqldb.util.QueryTool" archive="hsqldb.jar" width="460" height="400" codebase=".">
 <param name="database" value=".">
 </applet>
 <p>
--- hsqldb-1.8.0.0.orig/demo/servletdemo.html
+++ hsqldb-1.8.0.0/demo/servletdemo.html
@@ -2,7 +2,7 @@
 <head><title>HSQL Database Engine</title></head>
 <body bgcolor=#FFFFFF>
 <h2>HSQL Database Engine Servlet Demo</h2>
-<applet code="org.hsqldb.util.QueryTool" archive="hsqldb.jar" width="460" height="400" codebase="../lib/">
+<applet code="org.hsqldb.util.QueryTool" archive="hsqldb.jar" width="460" height="400" codebase=".">
 <param name="database" value="http://localhost:8080/examples/servlet/hsqlServlet">
 </applet>
 <p>
